source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -0,0 +1,321 @@
+Call Ingestion API
+=================
+
+The Call Ingestion API is used to submit call details along with a recording URL for calls that were not live-connected and routed through the Invoca Telephony platform.
+
+Calls submitted through this API are available to all post-call processing systems. This includes the Invoca transcription engine, Signal AI, as well as the Dashboard and Reporting system. 
+
+Invoca post-call processing services are applied to calls based on network and campaign settings.  The campaign for the call is determined by the *advertiser_campaign_id_from_network* parameter in the API.
+
+Additionally, Signals and Custom Data related to the call can be sumbitted during the initial call ingestion step.  Please note that Signals and Custom Data can also be applied in the future using the :doc:`../signal_api/index`.
+
+Signals can be any boolean value (e.g. sale, quote, etc), and Custom Data can be any alpha-numeric value (e.g. account type, customer quality score, etc).
+
+Response Codes
+--------------
+
+Remember to check the HTTP status code returned. This helps greatly when debugging.
+
+.. list-table::
+  :widths: 8 40
+  :header-rows: 1
+  :class: parameters
+
+
+  * - Status Code
+    - Meaning
+
+  * - 201 Created
+    - A new call creation request was successfully created.
+
+  * - 202 Accepted
+    - The call creation request has already been received and is currently being processed.
+
+  * - 400 Bad Request
+    - Attempted to make a request with an invalid API Version for route. Check the error message for any neccessary corrections
+
+  * - 401 Not Authorized
+    - Invalid or missing oauth token.
+
+  * - 403 Forbidden
+    - Attempted to access an invalid resource or provided invalid data. Check the errors object in the response.
+
+  * - 409 Conflict
+    - The **external_call_unique_id** in the request has already been used.  Please contact the Invoca support team at questions@invoca.com for further assistance.
+
+Request Parameters
+------------------
+
+Passed in “application/json” format.
+
+**Call Parameters**
+
+These are the call datails used when creating the call in the Invoca platform.
+
+    **Required**
+
+    `external_call_unique_id` The unique ID of the call in the system fom the external system.  This field is required to be unique across all calls within a network that are submitted from external sources.
```

Comment:
Nit: should we drop `in the system` here?

Reply:
agreed. updates pushed.